stan/math/prim/scal/prob/beta_binomial_rng.hpp

```diff
@@ -19,6 +19,40 @@
 namespace stan {
   namespace math {
 
+    /**
+     * Paranoid wrapper for beta_rng which enforces good output at the cost
```

Comment:
I'm actually going to cut this completely. if beta_rng actually returns values outside of [0,1] something is very wrong.